components/console-backend-service/internal/domain/k8s/kymaversion_resolver.go

```diff
@@ -0,0 +1,48 @@
+package k8s
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/k8s/pretty"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlerror"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/apps/v1"
+)
+
+//go:generate mockery -name=kymaVersionSvc -output=automock -outpkg=automock -case=underscore
+type kymaVersionSvc interface {
+	FindDeployment(name, namespace string) (*v1.Deployment, error)
+}
+
+//go:generate mockery -name=gqlKymaVersionConverter -output=automock -outpkg=automock -case=underscore
+type gqlKymaVersionConverter interface {
+	ToKymaVersion(in *v1.Deployment) string
+}
+
+type kymaVersionResolver struct {
+	kymaVersionSvc       kymaVersionSvc
+	kymaVersionConverter gqlKymaVersionConverter
+}
+
+func newKymaVersionResolver(kymaVersionSvc kymaVersionSvc) *kymaVersionResolver {
+	return &kymaVersionResolver{
+		kymaVersionSvc:       kymaVersionSvc,
+		kymaVersionConverter: &kymaVersionConverter{},
+	}
+}
+
+func (r *kymaVersionResolver) KymaVersionQuery(ctx context.Context) (string, error) {
+	name := "kyma-installer"
+	namespace := "kyma-installer"
+
+	deployment, err := r.kymaVersionSvc.FindDeployment(name, namespace)
```

Comment:
why not using a existing deployment service?
We dont need a dedicated `kymaVersionSvc` for that.. do we?